src/main/kotlin/no/nav/syfo/dialogmote/api/domain/TildelDialogmoterDTO.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.syfo.dialogmote.api.domain
+
+import java.util.UUID
+
+data class TildelDialogmoterDTO(
```

Comment:
Kjedelig med verb i klassenavn 😿

Reply:
Det er en request body, så usikker på hvordan vi ønsker å gjøre det her. Det er tydelig hvilken http request den tilhører.
Veldig åpen for alternativer så hvis du sitter inne med noen så fyr løs 🚀